resource_errors.json

```diff
@@ -0,0 +1,3 @@
+{
+    "topics": []
+}
```

Comment:
artifact, consider adding it to `.gitignore`

Reply:
Done!